examples/PT_afm_heisenberg.jl

```diff
@@ -0,0 +1,37 @@
+using Sunny
+
```

Comment:
Now that we have `@require MPI`, I think this script needs `using MPI` at the top to bring `Replica` and `run!` into scope.

Reply:
Done.